test/integration/patients/patient/sidebar.js

```diff
@@ -91,6 +91,7 @@ context('patient sidebar', function() {
     cy
       .server()
       .routePatientActions(_.identity, '2')
+      .routeForm()
```

Comment:
Since the form contents aren't interacted with in the test, the test wasn't broken by adding `this.form.fetch()` in the forms service.

But for the form to display in the snapshots and to remove the `400` request error, this route needed to be added.